src/modules/ethereum/chains/types.ts

```diff
@@ -0,0 +1,25 @@
+import type { NetworkConfigMetaData } from 'src/modules/networks/Networks';
+import type { NetworkConfig } from '@zeriontech/transactions/lib/shared/Networks.type';
+import type { AddEthereumChainParameter } from '../types/AddEthereumChainParameter';
+
+type DeprecatedEthereumChainConfig = NetworkConfigMetaData & {
+  value: NetworkConfig;
+};
+
+export interface EthereumChainConfig extends NetworkConfigMetaData {
+  value: AddEthereumChainParameter;
+  previousIds: string[];
```

Comment:
maybe it can be null?
```suggestion
  previousIds: string[] | null;
```